tests/autoyast/pxe_boot.pm

```diff
@@ -28,7 +28,9 @@ sub run {
 
     type_string "  ";    #need to separate default params
     type_string "vga=791 ";
-    type_string "Y2DEBUG=1 ";
+    if (get_var("Y2DEBUG")) {
+        type_string "Y2DEBUG=" . get_var("Y2DEBUG") . " ";
+    }
```

Comment:
shouldn't it be safe to just call

    type_string "Y2DEBUG=" . get_var("Y2DEBUG") . " ";